tcms/bugs/templates/bugs/get.html

```diff
@@ -162,12 +162,19 @@ <h2 class="card-pf-title">
                     </div>
 
                     <div class="form-group">
-                        <div>
-                            <button type="submit" name="action" value="comment" class="btn btn-default btn-lg">{% trans "Save" %}</button>
-
-                            <button type="submit" name="action" value="close" class="btn btn-danger btn-lg" style="float:right">
-                                {% trans "Close" %}
+                        <div style="padding:5px;display:flex; justify-content: space-between;">
+                            <button type="submit" name="action" value="comment" class="btn btn-default btn-lg">
+                                {% trans "Save" %}
                             </button>
+
+                            <div>
+                                <button type="submit" name="action" value="close" class="btn btn-warning btn-lg">
+                                    {% trans "Close" %}
+                                </button>
+                                <button type="button" name="action" value="delete" class="btn btn-danger btn-lg js-delete-bug">
+                                    {% trans "Delete" %}
+                                </button>
+                            </div>
```

Comment:
All of this goes in the edit menu of the page, not as buttons, to be consistent with other pages. And needs to use the delete view from the admin panel (which doesn't yet exist for bug models and will require some adjustments).